src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -269,7 +269,10 @@ private async Task UnsynchronizedStartHostAsync(ScriptHostStartupOperation activ
                 // If we're in a non-transient error state or offline, skip host initialization
                 bool skipJobHostStartup = isOffline || hasNonTransientErrors;
                 bool skipHostJsonConfiguration = startupMode == JobHostStartupMode.HandlingConfigurationParsingError;
-                _logger.Building(skipJobHostStartup, skipHostJsonConfiguration, activeOperation.Id);
+
```

Comment:
nit: remove the 2 unnecessary newlines on 272/274. Also use string instead of var below.